src/Core/Installer/Requirements/FilesystemRequirementsValidator.php

```diff
@@ -41,6 +42,13 @@ public function validateRequirements(RequirementsCheckCollection $checks): Requi
 
     private function existsAndIsWritable(string $path): bool
     {
+        if (!is_dir($path)) {
+            try {
+                (new Filesystem())->mkdir($path);
```

Comment:
This seems a bit random to me. A validator, which is clearly named after a simple check, creates directories as a side effect?

Reply:
yes :see_no_evil: I will clean this up, after all tests are green. for now I wanted to get things done :grimacing:

Reply:
I removed the check for `var\log`

see https://github.com/symfony/symfony/pull/58564